### PR TITLE
Added documentation for if elseif and else in go

### PR DESCRIPTION
**Pull request by piyushs2894**

### CODE REVIEW

File: if-elseif-else.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# If-Elseif-Else
+
+The example demonstrates usage of if, elseif and else clause in go.
+
+```
+package main
+
+import "fmt"
+
+/*
```
Comment (sagar-jadhav) [inline if-elseif-else.md]: Remove description from here

File: if-elseif-else.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+---
+
+# If-Elseif-Else
+
+The example demonstrates usage of if, elseif and else clause in go.
+
```
Comment (sagar-jadhav) [inline if-elseif-else.md:8]: Add description here

**Comment by sagar-jadhav** [comment]
Linking with #47

**Review by sagar-jadhav — CHANGES_REQUESTED** [review]
Add link to if-elseif-else.md in README as `if-elseif-else.html`

File: if-elseif-else.md (hunk 1 above)
Comment (piyushs2894) [reply if-elseif-else.md]: done.

File: if-elseif-else.md (hunk 2 above)
Comment (piyushs2894) [reply if-elseif-else.md:8]: done

**Review by sagar-jadhav — COMMENTED** [review]
Update link in READ me as follows:

`./if-elseif-else.html`

**Comment by piyushs2894** [comment]
Updated link in Readme

**Review by sagar-jadhav — APPROVED** [review]

**Pull request merged**